WcaOnRails/app/webpacker/components/RolesTab/DelegateForm.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import { Form } from 'semantic-ui-react';
+import I18n from '../../lib/i18n';
+
+const delegateStatusOptions = [{
+  text: I18n.t('enums.user.delegate_status.trainee_delegate'),
+  value: 'trainee_delegate',
+}, {
+  text: I18n.t('enums.user.delegate_status.candidate_delegate'),
+  value: 'candidate_delegate',
+}, {
+  text: I18n.t('enums.user.delegate_status.delegate'),
+  value: 'delegate',
+}, {
+  text: I18n.t('enums.user.delegate_status.senior_delegate'),
+  value: 'senior_delegate',
+}];
+
+const defaultValue = {
+  delegateStatus: delegateStatusOptions[0].value,
+  location: '',
+};
+
+export default function DelegateForm({
+  formValues,
+  setFormValues,
+  roleData,
+  seniorDelegates,
+}) {
+  React.useEffect(() => {
+    setFormValues({ ...defaultValue, ...roleData });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why are you disabling exhaustive deps?

Reply:
Since it was asking to add `setFormValues` to dependencies. But I've now changed it to handle in parent class. So this line is no longer there.